CODEOWNERS

```diff
@@ -130,6 +130,7 @@
 /drivers/flash/*spi_nor*                  @pabigot
 /drivers/flash/*stm32*                    @superna9999
 /drivers/gpio/*ht16k33*                   @henrikbrixandersen
+/drivers/gpio/*lmp90xxx*                  @henrikbrixandersen
```

Comment:
Shouldn't this be `/drivers/adc/*lmp90xxx`?

Reply:
@gon1332 No. All ADC drivers are owned by @anangl.